packages/disputer/src/disputer.js

```diff
@@ -227,6 +232,22 @@ class Disputer {
       return;
     }
 
+    // In legacy versions of the EMP, withdrawing needs to be done by a party involved in the liquidation (i.e liquidator,
+    // sponsor or disputer). As the disputer is the DSProxy, we would require the ability to send the withdrawal tx
+    // directly from the DSProxy to facilitate this. This functionality is not implemented as almost all legacy EMPs expired.
+    if (
+      this.proxyTransactionWrapper?.useDsProxyToDispute &&
+      this.isLegacyEmpVersion &&
+      disputedLiquidations.length > 0
+    ) {
+      this.logger.warn({
+        at: "Disputer",
+        message: "Attempting to withdraw dispute from a legacy EMP🙈",
+        details: "This is not supported on legacy with a DSProxy! Please manually withdraw the dispute",
```

Comment:
Should we include additional information here so we can identify the liquidations in question?